Dockerfile

```diff
@@ -69,7 +69,7 @@ RUN rm -rf postgres_install/build && \
     cd postgres_install && tar -czf /data/postgres_install.tar.gz . && cd .. && \
     rm -rf postgres_install
 
-RUN useradd -m -d /data zenith
+RUN useradd -m -d /data zenith && chown -R zenith:zenith /data
```

Comment:
It seems that `-m` isn't needed that way.

I also wonder how volume mounting works here. Directory `/data` exists and it isn't empty, but when we mount a directory from host to `/data` these already existing files are not visible outside Docker

Reply:
> I also wonder how volume mounting works here.

I don't know about macOS, but on Linux there's a thing called `bind mount` which allows you to put one directory on top of the other, regardless of the contents of the latter:

```sh
mkdir /tmp/foo
touch /tmp/foo/dummy
sudo mount --bind /tmp /tmp/foo
```

```
$ mount
...
tmpfs on /tmp/foo type tmpfs (rw,nosuid,nodev,size=8054572k,nr_inodes=409600,inode64)
```

Reply:
To support my claim, this is how what it looks like from within the container with a volume:

```
$ mount | grep /data
tmpfs on /data type tmpfs (rw,nosuid,nodev,size=8054572k,nr_inodes=409600,inode64)
```

and without one:

```
$ mount | grep /data
/dev/nvme0n1p3 on /data type ext4 (rw,relatime,data=ordered)
```

Reply:
> It seems that -m isn't needed that way.

Sure, I dropped it.